app/src/main/java/com/github/polypoly/app/game/Localization.kt

```diff
@@ -1,20 +1,44 @@
 package com.github.polypoly.app.game
 
+import org.osmdroid.util.GeoPoint
+
+/**
+    A Zone represents a group of [Localization]s, which share a common [color].
+    @property localizations a list of [Localization]s that belong to this zone.
+    @property color the color used to represent this zone.
+ */
+
 class Zone(
     val localizations: List<Localization>,
     val color: Int //> hexadecimal representation
-){}
+)
 
+/**
+    An enum class representing the different levels of a [Localization].
+    @property EMPTY the lowest level of a [Localization], when it is not yet owned.
+    @property STUDY_ROOM a level of a [Localization], when it has been acquired by the player.
+    @property AMPHITHEATER a level of a [Localization], when it has been upgraded once.
+    @property ARENA a level of a [Localization], when it has been upgraded twice.
+    @property STADIUM the highest level of a [Localization], when it has been upgraded thrice.
+ */
 enum class LocalizationLevel {
     EMPTY, STUDY_ROOM, AMPHITHEATER, ARENA, STADIUM
```

Comment:
I like the idea very much, but I find that Arena and Stadium are no longer related to EPFL and I think we can do better. I propose instead:
- EMPTY
- STUDY_ROOM
- AMPHUTHEATER
- FLOOR
- BUILDING

Reply:
While I totally agree, I think Floor could be taken out of context and would prefer replacing FLOOR by BUILDING directly, and BUILDING could then become LEARNING CENTER or similar (as it does not make much to go from amphitheater to a floor, a floor would already be existing already in order for the amphitheater to be there supposedly). Another suggestion would be Library or Laboratory

Reply:
I would go for LEVEL_0, LEVEL_1, LEVEL_2, LEVEL_3 and LEVEL_MAX as the actual meaning in game-terms should be taken care of by the front-end. This would also let us handle multiple languages and concentrate on the inner meaning of this enum (it represents the possible levels of a localization), and not how we interpret them (is level 1 a study room or an exercise room in our game rp? We don't care, it is the second level of evolution)

Reply:
Sounds good to me